remotespark/livyclientlib/livysession.py

```diff
@@ -68,16 +68,20 @@ def start(self):
 
         self.logger.debug("Session '{}' started.".format(self.kind))
 
-    def create_sql_context(self):
+    def create_sql_context(self, msg_printer):
         """Create a sqlContext object on the session. Object will be accessible via variable 'sqlContext'."""
         if self.started_sql_context:
             return
 
         self.logger.debug("Starting '{}' sql and hive session.".format(self.kind))
 
+        msg_printer.print_message('Creating SQL Context...')
```

Comment:
Also, what about adding some output to `wait_for_idle`? I was thinking maybe adding some dots every time the method is run and it's not in idle, so it'd look something like:

```
Creating SqlContext as 'sqlContext'.
.....
Creating HiveContext as 'hiveContext'.
.........
```

and for execute methods it'd look something like:
```
.....................
res0: DataFrame[column1, column2]
```